### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by etokareva**
Adaptive menu

### CODE REVIEW

File: themes/qgrid/source/css/icons.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@import "variables";
```
Comment (klumba12) [inline themes/qgrid/source/css/icons.scss]: use singular form for file names `icon.scss`

File: themes/qgrid/layout/layout.ejs [hunk 2]
```diff
@@ -47,32 +49,41 @@
...
+<header>
+    <%- partial('partials/header') %>
+</header>
 
-    <nav>
-      <%- partial('partials/nav') %>
-    </nav>
+<aside id="nav">
+    <div class="nav-header">
+        <button class="hamburger nav-trigger"></button>
```
Comment (klumba12) [inline themes/qgrid/layout/layout.ejs]: nav-trigger is not enought? do we need hamburger? )

File: themes/qgrid/source/css/icons.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+.icon {
+  display: inline-block;
```
Comment (klumba12) [inline themes/qgrid/source/css/icons.scss]: is is settings for header icons if so maybe good idea to put appropriate style to head itself?

File: themes/qgrid/source/css/layout-page.scss [hunk 4]
```diff
@@ -0,0 +1,104 @@
+@import "variables";
```
Comment (klumba12) [inline themes/qgrid/source/css/layout-page.scss]: what does layout-page mean? name is not obvious, if this is only for pages please call it `page.scss`

File: themes/qgrid/source/js/menu.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+function toggleDisplay(e) {
```
Comment (klumba12) [inline themes/qgrid/source/js/menu.js]: rename to toggleVisibility

File: themes/qgrid/source/js/menu.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+			li[i].style.display = "";
+		} else {
+			li[i].style.display = "none";
+		}
+	}
+}
+
+function init() {
+
+	var triggerBtns = document.getElementsByClassName("nav-trigger");
```
Comment (klumba12) [inline themes/qgrid/source/js/menu.js]: i believe we are going to support only 1 trigger button :)

File: themes/qgrid/layout/layout.ejs (hunk 2 above)
Comment (etokareva) [reply themes/qgrid/layout/layout.ejs]: @klumba12 I wanted to separate js-classes from styles :-)  We can name all our js-classes with 'js-' prefix. What do you think?

File: themes/qgrid/source/js/menu.js (hunk 6 above)
Comment (etokareva) [reply themes/qgrid/source/js/menu.js]: Yes, but I had to place them in different templates (for nav and for header). They won't be visible togeather at the same time

File: themes/qgrid/source/css/icons.scss (hunk 3 above)
Comment (etokareva) [reply themes/qgrid/source/css/icons.scss]: I'll remove grid-area from icons - styles to meke them more common

File: themes/qgrid/layout/layout.ejs (hunk 2 above)
Comment (klumba12) [reply themes/qgrid/layout/layout.ejs]: this is not js class, the name is obvious enough, there is no sence to have both classes from mine standpoint

File: themes/qgrid/source/js/menu.js (hunk 6 above)
Comment (klumba12) [reply themes/qgrid/source/js/menu.js]: ok this is make sence can you add a small comment about this, please

File: themes/qgrid/layout/layout.ejs (hunk 2 above)
Comment (etokareva) [reply themes/qgrid/layout/layout.ejs]: 👌 🆗

**Review by klumba12 — APPROVED** [review]

**Pull request merged**